ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -141,7 +141,7 @@ openstack*)
     source "${SHARED_DIR}/cinder_credentials.sh"
     export TEST_PROVIDER='{"type":"openstack"}';;
 ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
-equinix-ocp-metal)
+equinix-ocp-metal*)
```

Comment:
Hi @kuiwang02  can you (dis)confirm we need this and if needed add a /approve for this file's changes?

The cluster type being used for bm clusters is `equinix-ocp-metal-qe`, but a `equinix-ocp-metal` was previously used and exists.

Reply:
@aleskandro thanks for the notice.
I suggest `equinix-ocp-metal|equinix-ocp-metal-qe)` to restrict it exactly in case there is other `equinix-ocp-metal-xxx` which is not we want.

Reply:
Fixed @kuiwang02 (note there is another glob-based case for OpenStack a few rows above, probably to fix as well).